pdns/webserver.cc

```diff
@@ -308,17 +308,54 @@ try {
   resp.write(ss);
   string reply = ss.str();
 
+  ComboAddress remote;
+  client->getRemote(remote);
+  L<<Logger::Warning<<d_logprefix<<remote<<" \""<<req.method<<" "<<req.url.path<<" HTTP/"<<req.versionStr(req.version)<<"\" "<<resp.status<<" "<<reply.size()<<endl;
```

Comment:
Logging this at warning might be a bit too verbose? Perhaps we should hide that behind a specific flag?

Reply:
Perhaps we should log this line at the NOTICE level?